### PR TITLE
Edit the workflow for the SUSE install test

### PR DESCRIPTION
**Pull request by tannoa2**
@RackHD/doccommitters @jlongever @keedya @zyoung51 @jlongever

### CODE REVIEW

**Comment by yyscamper** [comment]
+1

**Comment by jlongever** [comment]
test this please

File: test/tests/api/v1_1/os_install_tests.py [hunk 1]
```diff
@@ -134,9 +134,9 @@ def install_suse(self, version, nodes=[], options=None):
                 'options': {
                     'defaults': {
                         'installDisk': '/dev/sda',
-                        'kvm': 'undefined', 
                         'version': version,
-                        'repo': os_repo
+                        'repo': os_repo,
+			'users': [{ 'name': 'onrack', 'password': 'Onr@ck1!', 'uid': 1010 }]
```
Comment (benbp) [inline test/tests/api/v1_1/os_install_tests.py]: should be spaces instead of tabs here.

**Comment by tannoa2** [comment]
test this please

**Pull request merged**